### PR TITLE
Create UpCloud partner page

### PR DESCRIPTION
**Pull request by isarpila**
Created this as I it the following instructions from: @pajuric

will create a new issue next with detailed information to this new partner page

Thank you for reaching out. Getting started as a partner is fairly easy, it's a matter of opening a PR in GitHub following the directions here: https://opensearch.org/new-partner.html. At the same time, you are welcome to submit for a partner page by using the information in this issue: https://github.com/opensearch-project/project-website/issues/1345 to create a new issue

### Description
[Describe what this change achieves]
 
### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by pajuric** [comment]
[UpCloud_brand_assets (3).zip](https://github.com/opensearch-project/project-website/files/11490942/UpCloud_brand_assets.3.zip)

File: _community_projects/upcloud
Comment (krisfreedain) [inline _community_projects/upcloud:1]: Hello @isarpila - thank you for submitting the PR! I believe you meant to be listed as a partner, not a community project. 

please submit your '**upcload.md**' file under the **_partners** directory
please submit the **logo** of your choosing under the **assets/media/partners**/ directory
(the image needs to be square to show properly)

thank you

Comment (pajuric) [reply _community_projects/upcloud:1]: @isarpila - Please use this link: https://opensearch.org/new-partner.html

Comment (isarpila) [reply _community_projects/upcloud:1]: @pajuric that link creates a page to _community_projects (can you verify if the link works as it should?)

image

Comment (nateynateynate) [reply _community_projects/upcloud:1]: > @pajuric that link creates a page to _community_projects (can you verify if the link works as it should?)
> 

Can confirm actually - the "create PR" button at the bottom is linking to the wrong spot. Will file an issue.

Comment (pajuric) [reply _community_projects/upcloud:1]: Thanks for addressing that, Nate.

**Comment by isarpila** [comment]
Can you maybe move this to the right place to get the partner page started? As the link is not working yet. Thanks @pajuric & co!

**Comment by isarpila** [comment]
Double checking if UpCloud partner page goes forward or if I need to make a new request for that?

**Comment by krisfreedain** [comment]
@isarpila  Which exact logo do you want to use? Let me know and I'll take care of the rest.

**Comment by krisfreedain** [comment]
@isarpila there are 9 different logos in that file. Which exact logo should we use? (the image needs to be square to show properly)

**Comment by isarpila** [comment]
ok can you use file named upcloud_logo.png then.

**Comment by krisfreedain** [comment]
helping resolve this in PR https://github.com/opensearch-project/project-website/pull/1626

**Pull request closed without merging**